src/data/full-quiz.js

```diff
@@ -1,4 +1,13 @@
 const fullQuiz = [
+  {
+    Question: "Which comparison operator is used to express strict equality, meaning that two compared values have the same type and equal value?",

```

Comment:
For the question, I would like to shorten it and mention JavaScript.

Proposed fix: Which comparison operator is used to express strict equality in JavaScript?

Reply:
Sounds good